### PR TITLE
Run tests as if the computer was in Copenhagen

### PR DESCRIPTION
**Pull request by carlbordum**
The tests can now run *everywhere*!

(disclaimer: I only verified that they work in Copenhagen, Danmarkshavn (Grønland) and Mexico)

### CODE REVIEW

File: oio_rest/oio_rest/utils/test_support.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ def _initdb():
                 settings.DATABASE, settings.DB_PASSWORD,
             ))
 
+            curs.execute("ALTER DATABASE {} SET time zone 'Europe/Copenhagen'".format(settings.DATABASE))
```
Comment (danchr) [inline oio_rest/oio_rest/utils/test_support.py]: We should add a comment prior to this line explaining why it's necessary, and it's not part of the templates. The obvious reason is that we do not, in fact, want to hard-code a time zone in production.

File: oio_rest/tests/test_integration_activity.py [hunk 2]
```diff
@@ -478,7 +479,8 @@ def test_searching_with_limit_after_editing_bvn(self):
     def test_searching_temporal_order(self):
         objids = [str(uuid.UUID(int=i)) for i in range(3)]
 
-        no_time = datetime.datetime.now()
+        tz = dateutil.tz.gettz('Europe/Copenhagen')
+        no_time = datetime.datetime.now(tz).replace(tzinfo=None)
```
Comment (danchr) [inline oio_rest/tests/test_integration_activity.py]: Move to a utility method?

File: oio_rest/tests/test_integration_activity.py [hunk 3]
```diff
@@ -497,7 +499,7 @@ def test_searching_temporal_order(self):
 
         time.sleep(0.01)
 
-        exists_time = datetime.datetime.now()
+        exists_time = datetime.datetime.now(tz).replace(tzinfo=None)
```
Comment (danchr) [inline oio_rest/tests/test_integration_activity.py]: Ditto?

File: oio_rest/tests/test_integration_activity.py (hunk 2 above)
Comment (carlbordum) [reply oio_rest/tests/test_integration_activity.py]: I don't believe in abstractions in the test suite :p

**Pull request merged**